awswrapper/s3.go

```diff
@@ -45,6 +56,14 @@ func GetS3Service(region string) *S3Service {
 	return s3Service
 }
 
+func (o *S3Service) GetSession() *ServiceSession {
+	return &ServiceSession{
+		Sess: session.Must(session.NewSession(&aws.Config{
+			Region: aws.String(o.region),
+		})),
+	}
+}
```

Comment:
Why do we need this? We already have `sess` var available which is initialized using init.

Reply:
In other functions, session is initialized for each call. I made this function so that calling code can get hold of session and pass it around various places (in both upload and download)

Reply:
Then we can name this function GetNewSession